README.md

```diff
@@ -74,6 +77,9 @@ giniConfiguration.backgroundColor = UIColor.white
 GiniVision.setConfiguration(giniConfiguration)
 ```
 
+## iPad support
+On iPad, unlike iPhone devices, the Gini Vision Library UI supports both landscape and portrait orientations, being full rotation enabled by default in both API's (*Screen* and *Component*). So in case you don't want one of the orientations, you need to disable it in your project settings (*Project Settings &rarr; General &rarr; Deployment Info &rarr; Device orientation*). 
```

Comment:
On iPad, unlike the iPhone, the Gini Vision Library UI supports both landscape and portrait orientations, full rotation being enabled by default in both APIs (*Screen* and *Component*). So in case you don't want any orientation, you need to disable it in your project settings (*Project Settings &rarr; General &rarr; Deployment Info &rarr; Device orientation*).